web/src/app/orbs/inhibit/page.tsx

```diff
@@ -0,0 +1,31 @@
+import OrbLayout from "../components/orb-layout/orb-layout";
+
+export default function InhibitPage() {
+  return (
+    <OrbLayout
+      title="Inhibit"
+      description="Disables the power absorbing mechanism of your opponent's pieces. While they can still swap powers, this inhibits their ability to acquire new powers by docking with the Power Orb. Available in three variants: row, column, radial."
```

Comment:
I'd change it to say that they can still use the powers they possess instead of swap.